README.rst

```diff
@@ -42,6 +42,22 @@ From the command-line, this requires some configuration::
     key = cloudstack api key
     secret = cloudstack api secret
 
+The config can also specify no ssl verification:
+    cat $HOME/.cloudstack.ini
+    [cloudstack]
+    endpoint = https://api.exoscale.ch/compute
+    key = cloudstack api key
+    secret = cloudstack api secret
+    verify = False
```

Comment:
Sure I get your point. I was just in the situation where we did not have a valid certificate on the server for some reason. It was in a lab/test situation using ansible. It has since yesterday been resolved. I just compared to cloudmonkey and other where verify ssl argument/config is generally an option. The thought about my pull request was mimic the use requests lib argument. Just specifying a cert can be done by pointing the REQUESTS_CA_BUNDLE at the ca-cert. With that I mean that this pull request can be dismissed if you only want to be able to point out a ca-cert, and just add REQUESTS_CA_BUNDLE to the README. :-) I'll look into if there is any support for client cert in the request library!